docs/source/releasing.rst

```diff
@@ -37,31 +37,18 @@ Things to do for releasing:
 
 * create pull request to ``python-trio/trio``'s "master" branch
 
-* announce PR on gitter
-
-   + wait for feedback
-
-   + fix problems, if any
-
 * verify that all checks succeeded
 
-* acknowledge the release PR
-
-   + or rather, somebody else should do that
-
-* tag with vVERSION
+* tag with vVERSION, push tag
 
 * push to PyPI
 
    + ``python3 setup.py sdist bdist_wheel upload``
```

Comment:
I tend to do:

```
git clean -xdf   # maybe run 'git clean -xdn' first to see what it will delete
python3 setup.py sdist bdist_wheel
twine upload dist/*
```

Historically `twine` was better than `setup.py upload`, but I don't know if that's still true.

Reply:
This is what I use too! Updated